src/lib/mongoBackend/mongoUpdateContextSubscription.cpp

```diff
@@ -382,6 +382,7 @@ HttpStatusCode mongoUpdateContextSubscription
 
     //
     // Update sub-cache
+    // FIXME PR: this is safe without sem?
```

Comment:
Issue created (and FIXME PR properly changed in the code): https://github.com/telefonicaid/fiware-orion/issues/2146